src/main/java/org/mandas/docker/client/AnsiProgressHandler.java

```diff
@@ -46,7 +45,7 @@ public AnsiProgressHandler() {
 
   public AnsiProgressHandler(PrintStream out) {
     this.out = out;
-    idsToLines = Maps.newHashMap();
+    idsToLines = new HashMap<>();
```

Comment:
seems like this should be `idsToLines = new HashMap<String, Integer>();` in order to avoid a warning?